src/modifiers/surrounding_pair.py

```diff
@@ -8,7 +8,6 @@
 """
 
 pair_symbols = {
-    "angle": "angleBrackets",
```

Comment:
this was actually just causing problems from previous merge; "angle" is also the name of the char in my repo

Reply:
Couldn't we just rename it to diamond? I think we probably should synchronize wrap and pair commands.

Reply:
See the next line 😊